CHANGELOG.md

```diff
@@ -1,3 +1,564 @@
+  ([#646](https://github.com/kinvolk/lokomotive/pull/646)).
+
+- Add admission webhook to lokomotive, which disables automount of `default` service account token
+  ([#704](https://github.com/kinvolk/lokomotive/pull/704)).
+
+- [Breaking Change] Kubelet joins cluster using TLS Bootstrapping now, add flag
+  `enable_tls_bootstrap = false` to disable.
+  ([#618](https://github.com/kinvolk/lokomotive/pull/618)).
+
+- Add `csi_plugin_node_selector` and `csi_plugin_toleration` for rook-ceph's CSI plugin
+  ([#892](https://github.com/kinvolk/lokomotive/pull/892)).
+
+
+#### Docs
+
+- Setting up third party OAuth for Grafana ([#542](https://github.com/kinvolk/lokomotive/pull/542)).
+
+- Upgrading bootstrap kubelet ([#592](https://github.com/kinvolk/lokomotive/pull/592)).
+
+- Upgrading etcd ([#802](https://github.com/kinvolk/lokomotive/pull/802)).
+
+- How to add custom monitoring resources? ([#554](https://github.com/kinvolk/lokomotive/pull/554)).
+
+- Kubernetes storage with Rook Ceph on Packet cloud
+  ([#494](https://github.com/kinvolk/lokomotive/pull/494)).
+
+
+#### Bug fixes
+
+- aws: Add check for multiple worker pools with same LB ports
+  ([#889](https://github.com/kinvolk/lokomotive/pull/889)).
+
+- packet: ignore changes to plan and user_data on controller nodes
+  ([#907](https://github.com/kinvolk/lokomotive/pull/907)).
+
+- Introduce platform.PostApplyHook interface and implement it for AKS cluster
+  ([#886](https://github.com/kinvolk/lokomotive/pull/886)).
+
+- aws-ebs-csi-driver: add NetworkPolicy allowing access to metadata
+  ([#865](https://github.com/kinvolk/lokomotive/pull/865)).
+
+- pkg/components/cluster-autoscaler: fix checking device uniqueness
+  ([#768](https://github.com/kinvolk/lokomotive/pull/768)).
+
+#### Development
+
+- Replace use of github.com/pkg/errors.Wrapf with fmt
+  ([#831](https://github.com/kinvolk/lokomotive/pull/831),
+  [#877](https://github.com/kinvolk/lokomotive/pull/877)).
+
+- Refactor assets handling ([#807](https://github.com/kinvolk/lokomotive/pull/807)).
+
+- cli/cmd: improve --kubeconfig-file flag help message formatting
+  ([#818](https://github.com/kinvolk/lokomotive/pull/818)).
+
+- Use host's /etc/hosts entries for bootkube
+  ([#409](https://github.com/kinvolk/lokomotive/pull/409)).
+
+- Refactor Terraform executor ([#794](https://github.com/kinvolk/lokomotive/pull/794)).
+
+- Pass kubeconfig content around rather than a file path
+  ([#631](https://github.com/kinvolk/lokomotive/pull/631)).
+
+
+### Upgrading from v0.3.0
+
+#### Lokoctl Host binary upgrades
+
+##### terraform-provider-ct
+
+- Update the `ct` Terraform provider to `v0.6.1`, find the install instructions
+  [here](https://github.com/poseidon/terraform-provider-ct/blob/v0.6.1/README.md#install).
+
+#### Configuration syntax changes
+
+##### Disable TLS Bootstrap
+
+Disable TLS bootstrapping to avoid recreation of cluster. Add the following attribute to the
```

Comment:
```suggestion
In this release we added TLS bootstrapping support by default. To avoid cluster recreation, disable it by adding the following attribute to the
```